### DISCUSSION TITLE
How to obtain the strain data (like axial strain, radial strain, hoop strain) by setting Outputs section?

### DISCUSSION THREAD

**Discussion by gin199502** [GEOS-DEV/GEOS#2805] [Q&A] [answered]
Hello everyone,
I performed several simulations using a PoroelasticitySolver solver. I want to output the strain to analyze. 
What should I do?

**Comment by paveltomin** [comment]
GEOS don't store the strain, so there is no field to visualize. The only way is to compute it as a postprocessing in paraview from the displacement field gradients.

**Comment by herve-gross** [comment]
@sytuannguyen maybe you can help

**Comment by zhishuaizhang** [comment, marked as answer]
As mentioned by @paveltomin, probably the easiest way is to calculate strain in ParaView. Use the PythonCalculator of ParaView. The expression of the PythonCalculator should be something like 'strain( totalDisplacement)'.

**Reply by gin199502** [reply]
@paveltomin @herve-gross @zhishuaizhang Thank you for your reply!
Yes, it is 'strain(totalDisplacement)' in ParaView.
